helm/values.yaml

```diff
@@ -32,7 +32,13 @@ coder:
 
   # coder.annotations -- The Deployment annotations. See:
   # https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
-  annotations: {}
+  annotations:
+    imageregistry: "https://hub.docker.com/"
```

Comment:
I'm a bit concerned about us adding this new default, but I'll admit I don't entirely understand the behavior.

Also, our coder pod is hosted in GHCR. Does this impact the cluster's pull behavior for this pod?

Reply:
@bpmct sorry, i pushed these in by mistake. was using the annotations to test the indentation, syntax. my most recent commit removes them.